presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add :func:`approx_most_frequent` aggregation function.
+* Add support to :func:`truncate` function to allow the users specify the number of digits to the right of the decimal point in the truncated result.
+* Add support for ignoring access checks on columns that are referenced in the query, but are not required to compute the query results. This can be enabled with the ``check_access_control_on_utilized_columns_only`` session property.
+* Add support to verify ``CREATE VIEW`` and ``CREATE TABLE`` queries.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix query failure when explaining ``USE`` statement.
```

Comment:
This can be made clearer by saying - "Fix query failure when using ``EXPLAIN`` with a ``USE`` statement" . The word "explaining" is a bit unclear in my opinion